packages/dev/docs/pages/react-spectrum/ssr.mdx

```diff
@@ -128,17 +129,20 @@ With this installed, add the following to your `next.config.js` file.
 const withTM = require("next-transpile-modules")([
   "@adobe/react-spectrum",
   "@react-spectrum/actiongroup",
-  "@react-spectrum/breadcrumbs",
+  "@react-spectrum/avatar",
   "@react-spectrum/badge",
+  "@react-spectrum/breadcrumbs",
   "@react-spectrum/button",
   "@react-spectrum/buttongroup",
   "@react-spectrum/calendar",
   "@react-spectrum/checkbox",
+  "@react-spectrum/color",
```

Comment:
i don't see color in the monorepo?

Reply:
Good catch. Interesting that the config still works though. I'll remove this.

Reply:
As a sanity check, I confirmed that it still works without it.

Reply:
I think it just tells what nextjs to do if it encounters those modules? so if you're using any of our beta/alpha/rc etc packages, you'd need to add them here